### PR TITLE
feat(api): add POST v2/broadcast 

### PR DESCRIPTION
**Pull request by enddynayn**
Add v2/broadcast endpoint to submit a broadcast message
that gets processed by the Content-Publisher service.
This approach simplifies the process for developers by abstracting the 
complex details involved in directly interacting with the Frequency Chain.

#9

### CODE REVIEW

**Comment by enddynayn** [comment]
I think there are probably better ways to consume content-publisher open API spec. Ill open a ticket to look at what are the best practices for this.

File: backend/repositories/ContentPublisherRepository.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { Client as ContentPublisherClient } from "../types/openapi-content-publisher.js";
+import type { Components, Paths } from "../types/openapi-content-publisher.js";
+import OpenAPIClientAxios from "openapi-client-axios";
+export class ContentPublisherRepository {
+  private static instance: ContentPublisherRepository;
+  private _client: ContentPublisherClient;
```
Comment (claireolmstead) [inline backend/repositories/ContentPublisherRepository.ts:6]: whats the difference between _client and client? Purpose of "_"?

File: backend/repositories/ContentPublisherRepository.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    if (this._client === undefined) {
+      throw new Error("API not initialized");
+    }
+
+    return this._client;
+  }
+
+  public async postBroadcast(
+    msaId: string,
+    data: Components.Schemas.BroadcastDto,
```
Comment (claireolmstead) [inline backend/repositories/ContentPublisherRepository.ts:41]: Semantic, but can we just import the needed schemas or unwrap them like `const {BroadcastDto, AnnouncementResponseDto, ReplyDto} = Components.Schemas;`

**Review by claireolmstead — APPROVED** [review]
LGTM

File: backend/repositories/ContentPublisherRepository.ts (hunk 1 above)
Comment (enddynayn) [reply backend/repositories/ContentPublisherRepository.ts:6]: To distinguish between the getter and setter functions
```
  private set client(api: ContentPublisherClient) {
    this._client = api;
  }

  private get client() {
    if (this._client === undefined) {
      throw new Error("API not initialized");
    }

    return this._client;
  }
  ```

File: backend/repositories/ContentPublisherRepository.ts (hunk 2 above)
Comment (enddynayn) [reply backend/repositories/ContentPublisherRepository.ts:41]: These are types so we are unable to desconstruct them in that way. But your right its not great for reading. I updated the code to make it easier to read and not have to go through the namespace.

**Pull request merged**